client/src/main/java/org/apache/rocketmq/client/ClientConfig.java

```diff
@@ -60,6 +60,8 @@
 
     private boolean useTLS = TlsSystemConfig.tlsEnable;
 
+    private int mqClientApiTimeout = 3 * 1000;
```

Comment:
Other places, such as MQAdminImpl.java, RemoteBrokerOffsetStore.java , also use fixed timeout, should they change to use this config?

Reply:
IMO, add another config item is better and more readable.